Jenkinsfile

```diff
@@ -2,12 +2,14 @@
 
 @Library('pipeline-library') _
 
-def img
+def image
+def ws_only_image
 
 node {
     stage('build') {
         checkout(scm)
-        img = buildApp(name: 'hypothesis/hypothesis')
+        image = buildApp(name: 'hypothesis/hypothesis')
+        ws_only_image = buildApp(name: 'hypothesis/hypothesis-ws', path='h/streamer')
```

Comment:
I am wondering if the way `path='h/streamer'` is being defined is correct syntax. Should it not be: `path:  'h/streamer'`?

Reply:
Yah... had my Python head on...